install_config/install/advanced_install.adoc

```diff
@@ -315,6 +315,34 @@ node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 ----
 ====
 
+Nodes have labels that often indicate their region. A convention is to set the
+`*region=infra*` label for the router and registry selector. For example:
```

Comment:
Nodes only receive labels today if you explicitly add them in the inventory using `openshift_node_labels` as described below.  `region=infra` is the default selector and if you create Nodes with that selector then the router and registry will be installed.